packages/sit-onyx/src/components/OnyxCalendarCell/OnyxCalendarCell.vue

```diff
@@ -24,6 +24,11 @@ const slots = defineSlots<{
 }>();
 
 const { densityClass } = useDensity(props);
+const handleHover = () => {
+  if (!props.disabled) {
+    emit("hovered");
+  }
+};
```

Comment:
Prefer to set event-listener dynamically instead of having a conditional in the handler.
That way the the event-listener won't even be created:


```suggestion
const hoverHandler = computed(() =>
  !props.disabled
    ? {
        onMouseenter: () => emit("hovered"),
        onFocusin: () => emit("hovered"),
      } satisfies HTMLAttributes
    : {},
);
```